app/scripts/modules/core/src/serverGroupManager/serverGroupManager.states.ts

```diff
@@ -20,23 +20,12 @@ module(SERVER_GROUP_MANAGER_STATES, [APPLICATION_STATE_PROVIDER])
       url: '/serverGroupManagerDetails/:provider/:accountId/:region/:serverGroupManager',
       views: {
         'detail@../insight': {
-          templateProvider: ['$templateCache', '$stateParams', 'versionedCloudProviderService',
-            ($templateCache: ITemplateCacheService,
-             $stateParams: StateParams,
-             versionedCloudProviderService: VersionedCloudProviderService) => {
-              return versionedCloudProviderService.getValue($stateParams.provider, $stateParams.accountId, 'serverGroupManager.detailsTemplateUrl').then(templateUrl =>
-                $templateCache.get(templateUrl)
-              );
-            }],
-          controllerProvider: ['$stateParams', 'versionedCloudProviderService',
-            ($stateParams: StateParams,
-             versionedCloudProviderService: VersionedCloudProviderService) => {
-              return versionedCloudProviderService.getValue($stateParams.provider, $stateParams.accountId, 'serverGroupManager.detailsController');
-            }],
-          controllerAs: 'ctrl'
+          component: ServerGroupManagerDetails,
```

Comment:
@danielpeach please test that this details view still renders properly (I think it's kubernetes?)